Makefile.am

```diff
@@ -942,14 +1022,57 @@ dictsource/yue_emoji:
 
 ##### mbrola:
 
-PHSOURCE_MBROLA = \
-	$(wildcard phsource/mbrola/*)
-
 mbrola: \
-	$(patsubst phsource/mbrola/%,espeak-ng-data/mbrola_ph/%_phtrans,$(PHSOURCE_MBROLA))
+	espeak-ng-data/mbrola_ph/af1_phtrans \
+	espeak-ng-data/mbrola_ph/ar1_phtrans \
+	espeak-ng-data/mbrola_ph/ar2_phtrans \
+	espeak-ng-data/mbrola_ph/ca_phtrans \
+	espeak-ng-data/mbrola_ph/cmn_phtrans \
+	espeak-ng-data/mbrola_ph/cr1_phtrans \
+	espeak-ng-data/mbrola_ph/cs_phtrans \
+	espeak-ng-data/mbrola_ph/de2_phtrans \
+	espeak-ng-data/mbrola_ph/de4_phtrans \
+	espeak-ng-data/mbrola_ph/de6_phtrans \
+	espeak-ng-data/mbrola_ph/de8_phtrans \
+	espeak-ng-data/mbrola_ph/ee1_phtrans \
+	espeak-ng-data/mbrola_ph/en1_phtrans \
+	espeak-ng-data/mbrola_ph/es_phtrans \
+	espeak-ng-data/mbrola_ph/es3_phtrans \
+	espeak-ng-data/mbrola_ph/es4_phtrans \
+	espeak-ng-data/mbrola_ph/fr_phtrans \
+	espeak-ng-data/mbrola_ph/gr1_phtrans \
+	espeak-ng-data/mbrola_ph/gr2_phtrans \
+	espeak-ng-data/mbrola_ph/grc-de6_phtrans \
+	espeak-ng-data/mbrola_ph/he_phtrans \
+	espeak-ng-data/mbrola_ph/hn1_phtrans \
+	espeak-ng-data/mbrola_ph/hu1_phtrans \
+	espeak-ng-data/mbrola_ph/ic1_phtrans \
+	espeak-ng-data/mbrola_ph/id1_phtrans \
+	espeak-ng-data/mbrola_ph/in_phtrans \
+	espeak-ng-data/mbrola_ph/ir1_phtrans \
+	espeak-ng-data/mbrola_ph/it1_phtrans \
+	espeak-ng-data/mbrola_ph/it3_phtrans \
+	espeak-ng-data/mbrola_ph/jp_phtrans \
+	espeak-ng-data/mbrola_ph/la1_phtrans \
+	espeak-ng-data/mbrola_ph/lt_phtrans \
+	espeak-ng-data/mbrola_ph/ma1_phtrans \
+	espeak-ng-data/mbrola_ph/mx1_phtrans \
+	espeak-ng-data/mbrola_ph/mx2_phtrans \
+	espeak-ng-data/mbrola_ph/nl_phtrans \
+	espeak-ng-data/mbrola_ph/nz1_phtrans \
+	espeak-ng-data/mbrola_ph/pl1_phtrans \
+	espeak-ng-data/mbrola_ph/pt1_phtrans \
+	espeak-ng-data/mbrola_ph/ptbr_phtrans \
+	espeak-ng-data/mbrola_ph/ptbr4_phtrans \
+	espeak-ng-data/mbrola_ph/ro1_phtrans \
+	espeak-ng-data/mbrola_ph/sv_phtrans \
+	espeak-ng-data/mbrola_ph/sv2_phtrans \
+	espeak-ng-data/mbrola_ph/tl1_phtrans \
+	espeak-ng-data/mbrola_ph/tr1_phtrans \
+	espeak-ng-data/mbrola_ph/us_phtrans \
+	espeak-ng-data/mbrola_ph/us3_phtrans \
+	espeak-ng-data/mbrola_ph/vz_phtrans
 
 espeak-ng-data/mbrola_ph/%_phtrans: phsource/mbrola/% src/espeak-ng
 	mkdir -p espeak-ng-data/mbrola_ph
-	ESPEAK_DATA_PATH=$(CURDIR) src/espeak-ng --compile-mbrola=phsource/mbrola/$*
-
-EXTRA_DIST += $(PHSOURCE_MBROLA)
+	ESPEAK_DATA_PATH=$(CURDIR) src/espeak-ng --compile-mbrola=phsource/mbrola/$*
```

Comment:
You have unrelated changes reversions here. You probably mis-merged master at some point,  we can have a look at that tomorrow

Reply:
certainly, Thanks for your feedback